python/podio/sio_io.py

```diff
@@ -20,8 +21,9 @@ def __init__(self, filename):
         """Create a reader that reads from the passed file.
 
         Args:
-            filename (str): File to open and read data from
+            filename (str or Path): File to open and read data from.
         """
+        filename = convert_to_str_paths(filename)
```

Comment:
```suggestion
        filename = convert_to_str_paths(filename)[0]
```
As `openFile` takes a single string.